be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -388,6 +389,8 @@ Status SinkBuffer::_try_to_send_rpc(const TUniqueId& instance_id, const std::fun
             LOG(WARNING) << err_msg;
         });
         closure->addSuccessHandler([this](const ClosureContext& ctx, const PTransmitChunkResult& result) noexcept {
+            // when _total_in_flight_rpc desc to 0, _fragment_ctx may be destructed
+            auto defer = DeferOp([this]() { --_total_in_flight_rpc; });
```

Comment:
If _total_in_flight_rpc reaches 0 and there is no notification, how other threads know _total_in_flight_rpc is already 0. Are other threads busy-looping to check if it is 0?

Reply:
poller thread will check it